### PR TITLE
updated to work with one.com

### PR DESCRIPTION
**Pull request by der-berni**
rev command not found on OpenWrt
CURL does not work, using wget
JSESSIONID replaced with OneSIDCrmAdmin
CSRF_G_TOKEN not needed

### CODE REVIEW

File: dnsapi/dns_one.sh [hunk 1]
```diff
@@ -4,48 +4,68 @@
 # one.com ui wrapper for acme.sh
 # Author: github: @diseq
 # Created: 2019-02-17
+# Fixed by: @der-berni
+# Modified: 2019-05-20
 #
-#     export ONECOM_User="username"
-#     export ONECOM_Password="password"
+#     export ONECOM_USER="username"
```
Comment (Neilpang) [inline dnsapi/dns_one.sh]: do not change the name from  `ONECOM_User`  to  `ONECOM_USER`.
The existing users will kill you.

Comment (der-berni) [reply dnsapi/dns_one.sh]: That was my fault. I misread.
https://github.com/Neilpang/acme.sh/pull/2098#discussion_r258946332

See my second commit.

File: dnsapi/dns_one.sh [hunk 2]
```diff
@@ -41,11 +57,15 @@ dns_one_add() {
   postdata="$postdata&password1=$ONECOM_Password"
   postdata="$postdata&loginTarget="
   #_debug postdata "$postdata"
-
+  
+  #CURL does not work
+  local tmp_USE_WGET=$ACME_USE_WGET
+  ACME_USE_WGET=1
```
Comment (Neilpang) [inline dnsapi/dns_one.sh]: No,  never select to use wget or curl in your script.
It's control by acme.sh

if curl does not work,  you can display an `_err` message to ask the user to use wget wih `--use-wget`

Comment (Neilpang) [reply dnsapi/dns_one.sh]: I still don't understand why curl is not working ?

Comment (der-berni) [reply dnsapi/dns_one.sh]: the response from curl is every time the login page.
the same results on other devices.
i will try to get more info

Comment (der-berni) [reply dnsapi/dns_one.sh]: Without the follow redirects (Option) "-L" it works with curl

Comment (der-berni) [reply dnsapi/dns_one.sh]: i fixed to work with curl

**Comment by Neilpang** [comment]
fix the CI errors first.

**Comment by Neilpang** [comment]
would you like to create an issue for bug report: 

https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide#10-please-create-a-new-issue-for-future-bugs

**Pull request merged**